frontend/src/app/features/metadata/component/book-metadata-center/metadata-viewer/metadata-viewer.component.scss

```
@@ -89,27 +89,27 @@
 }
 
 ::ng-deep .progress-incomplete .p-progressbar-value {
```

Stylelint (17.11.1)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)